action.yml

```diff
@@ -29,13 +32,22 @@ runs:
     - uses: adam7/platformsh-cli-action@v1.1
       with:
         token: ${{ inputs.cli-token }}
+    -
+      name: Set environment name
+      id: set_environment_name
+      shell: bash
+      run: |
+        ENVIRONMENT_NAME=${{ inputs.environment-name }}
+        [[ "$ENVIRONMENT_NAME" == "" ]] && ENVIRONMENT_NAME="$GITHUB_REF_NAME"
+        echo "::set-output name=url::$ENVIRONMENT_NAME"
```

Comment:
I am not very fond of this. I think we can handle this in the shell script itself quite robustly. See my other two suggestions.